### ISSUE TITLE
Get nan loss when training with the 2nd GPU?

### ISSUE THREAD

**Issue by liujiaxu-tju** [facebookresearch/maskrcnn-benchmark#303]
## ❓ Questions and Help
I want to train model only using the 2nd GPU when the 1st is being used by others. I use the following command  to do this, but then all the loss become nan. By the way, it runs just well when I use the 1st GPU.

`python tools/train_net.py --config-file e2e_faster_rcnn_X_101_32x8d_FPN_1x.yaml MODEL.DEVICE "cuda:1"`

**Comment by ranjiewwen** [comment]
maybe the learning rate should be small ; the default set for 8 GPUs .

**Comment by fmassa** [comment]
Hi,

You should instead use it as follows:

```
CUDA_VISIBLE_DEVICES=1 python tools/train_net.py --config-file ...
```
or
```
CUDA_VISIBLE_DEVICES=2,3 python tools/train_net.py --config-file ...
```

the `MODEL.DEVICE` option would be a solution as well, but you are probably facing the issue in https://github.com/facebookresearch/maskrcnn-benchmark/issues/74 that needs to be addressed before this can be used as is.

Closing as a duplicate of #74
